cupy/random/_generator_api.pyx

```diff
@@ -159,6 +159,75 @@ class Generator:
         # omitted args.
         return (<object>y).astype(dtype, copy=False)
 
+    def uniform(self, low=0.0, high=1.0, size=None, dtype=numpy.float64):
```

Comment:
How about returning `low + (high - low) * self.random()` simply?

Reply:
That was my initial idea, but the problem is that it doesn't match numpy's behavior.  numpy allows mismatched inputs, e.g. "low" can be a float and "high" can be an array of length 10, in which case the output would be an array of length 10.